http_logs/index.json

```diff
@@ -1,6 +1,6 @@
 {
   "settings": {
-    "index.number_of_shards": {{number_of_shards | default(5)}},
+    "index.number_of_shards": {{number_of_shards | default(1)}},
```

Comment:
This seemed useful for me when I was testing the amount of space that fields take up. I'm happy to remove it from this PR if we'd like.

Reply:
Yeah this will be a surprise to users relying on defaults, I think we should try to keep this default value.

Reply:
++